src/amp-ads/40_Experimental_Ads/Scrollbound_Video_Ad.html

```diff
@@ -0,0 +1,263 @@
+
+    /*
+     * Top level container: It fills the parent viewport
+     * that hosts the ad.
+     */
+    .ad-container {
+      font-family: "Roboto", sans-serif;
+      position: relative;
+      width: 100vw;
+      height: 100vh;
+      color: #FFFFFF;
+    }
+
+    /* Video Layer */
+    .video-container {
+      /**
+       * Specifies that right corner should be used
+       * as the anchor point when video is scaled down
+       * using scrollbound amp-animation.
+       */
+      transform-origin: calc(100% - 25px) 25px;
+      z-index: 2;
+    }
+
+    /* Video player: Click-to-player overlay */
+    .video-container .poster-image {
+      position: absolute;
+      z-index: 1;
+    }
+
+    .video-container .poster-image img {
+      object-fit: cover;
+    }
+
+    .video-container .play-icon {
+      position: absolute;
+      z-index: 3;
+      width: 100px;
+      height: 100px;
+      background-image: url(<%host%>/img/play-icon.png);
+      background-repeat: no-repeat;
+      background-size: 100% 100%;
+      /* Align to the middle */
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      cursor: pointer;
+      opacity: 0.9;
+    }
+
+    .video-container amp-video video {
+      object-fit: cover;
+    }
+
+    /* Content Layer */
+    .content-container {
+      z-index: 1;
+    }
+
+    /* Content Layer: Background image */
+    .background amp-img {
+      margin: -70px;
+    }
+
+    .background amp-img img {
+      object-fit: cover;
+    }
+
+    /* Content Layer: Title and logo */
+    .title {
+      position: absolute;
+      top: 5px;
+      padding-left: 15px;
+      font-weight: bold;
+    }
+
+    /* Content Layer: Footer and learn more button */
+    .footer {
+      position: absolute;
+      bottom: 0;
+      width: 100%;
+      padding: 5px 15px;
+      background-color: rgba(0, 0, 0, 0.8);
+      box-sizing: border-box;
+      font-size: 10px;
+      display: flex;
+      flex-direction: row;
+    }
+
+    .footer>* {
+      flex: 1;
+      align-self: center;
+    }
+
+    .footer .learn-more {
+      background-color: #2979ff;
+      padding: 5px 15px;
+      white-space: nowrap;
+      flex: 0;
+      font-size: 10px;
+      letter-spacing: 0.5px;
+      color: #fafafa;
+      text-decoration: none;
+    }
+
+    /* Generic CSS class to fill a parent */
+    .fill {
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+    }
+  </style>
+</head>
+
+<body>
+  <!-- ## Scrollbound Animations -->
+  <!--
+    There are two types of scrollbound animations that occur in this sample:
+    **scaling** of the video container and **parallax** movement of the background image.
+
+    [amp-animation](https://www.ampproject.org/docs/reference/components/amp-animation)
+    is an experimental AMP component that uses [Web Animations API](https://www.w3.org/TR/web-animations/)
+    to support both __time__ and __scroll__ based animations.
```

Comment:
Suggestion: show the amp-animation code first (here) and then explain what's going on. At the moment there is too much text before you can see the code.